mobile-save-for-later/conf/cfn.yaml

```diff
@@ -21,87 +21,6 @@ Resources:
       Threshold: 0
       Unit: Count
       MetricName: WriteThrottleEvents
```

Comment:
Some resources remain in the cloudformation (dynamodb, alarms), is that because we don't have available cdk definitions for these yet?

Reply:
We can port these across too, but I suggest that we do it in separate PRs (since it's not quite as straightforward as the deletions that are happening in this PR):

1. Remove the alarms once https://theguardian.atlassian.net/browse/LIVE-4204 is complete
2. Port the Dynamo table from CFN to TypeScript (this will be similar to https://github.com/guardian/presence-indicator/pull/129 and [these docs](https://github.com/guardian/cdk/blob/main/docs/stateful-resources.md) are likely to provide some helpful context)